### ISSUE TITLE
log in with seed phrase stuck at encrypting wallet

### ISSUE THREAD

**Issue by smk762** [KomodoPlatform/atomicDEX#90]
Phone: Samsung S10e
To replicate: at enter unlock wallet screen, click on "sign in with seed phrase" beneath the login button. Enter seed on next screen and click confirm. For some reason, it asks for a new password creation, enter that and confirm. Read the TOS, click next. Here is gets stuck.

ADB logs: 

```09-09 02:20:43.672 22438 19903 E flutter : [ERROR:flutter/lib/ui/ui_dart_state.cc(148)] Unhandled Exception: NoSuchMethodError: The setter 'isFastEncryption=' was called on null.
09-09 02:20:43.672 22438 19903 E flutter : Receiver: null
09-09 02:20:43.672 22438 19903 E flutter : Tried calling: isFastEncryption=
09-09 02:20:43.672 22438 19903 E flutter : #0      _DislaimerPageState._nextPage (package:komodo_dex/screens/authentification/dislaimer_page.dart:378)
09-09 02:20:43.672 22438 19903 E flutter : <asynchronous suspension>
09-09 02:20:43.672 22438 19903 E flutter : #1      _InkResponseState._handleTap (package:flutter/src/material/ink_well.dart:635)
09-09 02:20:43.672 22438 19903 E flutter : #2      _InkResponseState.build.<anonymous closure> (package:flutter/src/material/ink_well.dart:710)
09-09 02:20:43.672 22438 19903 E flutter : #3      GestureRecognizer.invokeCallback (package:flutter/src/gestures/recognizer.dart:182)
09-09 02:20:43.672 22438 19903 E flutter : #4      TapGestureRecognizer._checkUp (package:flutter/src/gestures/tap.dart:365)
09-09 02:20:43.672 22438 19903 E flutter : #5      TapGestureRecognizer.handlePrimaryPointer (package:flutter/src/gestures/tap.dart:275)
09-09 02:20:43.672 22438 19903 E flutter : #6      PrimaryPointerGestureRecognizer.handleEvent (package:flutter/src/gestures/recognizer.dart:467)
09-09 02:20:43.672 22438 19903 E flutter : #7      PointerRouter._dispatch (package:flutter/src/gestures/pointer_router.dart:75)
09-09 02:20:43.672 22438 19903 E flutter : #8      PointerRouter.route (package:flutter/src/gestures/pointer_router.dart:102)
09-09 02:20:43.672 22438 19903 E flutter : #9      GestureBinding.handleEvent (package:flutter/src/gestures/binding.dart:218)
09-09 02:20:43.672 22438 19903 E flutter : #10     GestureBinding.dispatchEvent (package:flutter/src/gestures/binding.dart:198)
09-09 02:20:43.672 22438 19903 E flutter : #11     GestureBinding._handlePointerEvent (package:flutter/src/gestures/binding.dart:156)
09-09 02:20:43.672 22438 19903 E flutter : #12     GestureBinding._flushPointerEventQueue (package:flutter/src/gestures/binding.dart:102)
09-09 02:20:43.672 22438 19903 E flutter : #13     GestureBinding._handlePointerDataPacket (package:flutter/src/gestures/binding.dart:86)
09-09 02:20:43.672 22438 19903 E flutter : #14     _rootRunUnary (dart:async/zone.dart:1136)
09-09 02:20:43.672 22438 19903 E flutter : #15     _CustomZone.runUnary (dart:async/zone.dart:1029)
09-09 02:20:43.672 22438 19903 E flutter : #16     _CustomZone.runUnaryGuarded (dart:async/zone.dart:931)
09-09 02:20:43.672 22438 19903 E flutter : #17     _invoke1 (dart:ui/hooks.dart:263)
09-09 02:20:43.672 22438 19903 E flutter : #18     _dispatchPointerDataPacket (dart:ui/hooks.dart:172)
09-09 02:20:43.672 22438 19903 E flutter : 
```

**Comment by tonymorony** [comment]
Thanks for report! Reproduced on my side by following same steps with exactly same crash and trace

**Comment by bitbybit** [comment]
I have the same problem on Xiaomi Mi MIX 2S

```
--------- beginning of main
09-11 12:03:39.564 27550 27550 E tform.atomicde: Not starting debugger since process cannot load the jdwp agent.
09-11 12:03:39.593 27550 27550 W tform.atomicde: JIT profile information will not be recorded: profile file does not exits.
09-11 12:03:39.593 27550 27550 I chatty  : uid=10221(com.komodoplatform.atomicdex) identical 1 line
09-11 12:03:39.593 27550 27550 W tform.atomicde: JIT profile information will not be recorded: profile file does not exits.
09-11 12:03:39.631 27550 27550 D OpenGLRenderer: Skia GL Pipeline
09-11 12:03:39.642 27550 27550 I Adreno  : QUALCOMM build                   : 8c3e34a, Ic446a5d9fc
09-11 12:03:39.642 27550 27550 I Adreno  : Build Date                       : 05/13/19
09-11 12:03:39.642 27550 27550 I Adreno  : OpenGL ES Shader Compiler Version: EV031.25.03.07
09-11 12:03:39.642 27550 27550 I Adreno  : Local Branch                     : 
09-11 12:03:39.642 27550 27550 I Adreno  : Remote Branch                    : refs/tags/AU_LINUX_ANDROID_LA.UM.7.3.R1.09.00.00.423.045
09-11 12:03:39.642 27550 27550 I Adreno  : Remote Branch                    : NONE
09-11 12:03:39.642 27550 27550 I Adreno  : Reconstruct Branch               : NOTHING
09-11 12:03:39.642 27550 27550 I Adreno  : Build Config                     : S P 6.0.7 AArch64
09-11 12:03:39.644 27550 27550 I Adreno  : PFP: 0x016ee180, ME: 0x00000000
09-11 12:03:39.646 27550 27550 I ConfigStore: android::hardware::configstore::V1_0::ISurfaceFlingerConfigs::hasWideColorDisplay retrieved: 0
09-11 12:03:39.646 27550 27550 I ConfigStore: android::hardware::configstore::V1_0::ISurfaceFlingerConfigs::hasHDRDisplay retrieved: 0
09-11 12:03:39.927 27550 27585 I OpenGLRenderer: Initialized EGL, version 1.4
09-11 12:03:39.927 27550 27585 D OpenGLRenderer: Swap behavior 2
09-11 12:03:39.931 27550 27566 I flutter : loadJsonCoinsDefault
09-11 12:03:40.034 27550 27566 I flutter : loadJsonCoinsDefault
09-11 12:03:40.038 27550 27566 I flutter : 1
09-11 12:05:13.261 27550 27550 W IInputConnectionWrapper: getTextBeforeCursor on inactive InputConnection
09-11 12:05:13.262 27550 27550 W IInputConnectionWrapper: getSelectedText on inactive InputConnection
09-11 12:05:13.262 27550 27550 W IInputConnectionWrapper: getTextAfterCursor on inactive InputConnection
09-11 12:05:44.233 27550 27566 E flutter : [ERROR:flutter/lib/ui/ui_dart_state.cc(148)] Unhandled Exception: NoSuchMethodError: The setter 'isFastEncryption=' was called on null.
09-11 12:05:44.233 27550 27566 E flutter : Receiver: null
09-11 12:05:44.233 27550 27566 E flutter : Tried calling: isFastEncryption=
09-11 12:05:44.233 27550 27566 E flutter : #0      _DislaimerPageState._nextPage (package:komodo_dex/screens/authentification/dislaimer_page.dart:378)
09-11 12:05:44.233 27550 27566 E flutter : <asynchronous suspension>
09-11 12:05:44.233 27550 27566 E flutter : #1      _InkResponseState._handleTap (package:flutter/src/material/ink_well.dart:635)
09-11 12:05:44.233 27550 27566 E flutter : #2      _InkResponseState.build.<anonymous closure> (package:flutter/src/material/ink_well.dart:710)
09-11 12:05:44.233 27550 27566 E flutter : #3      GestureRecognizer.invokeCallback (package:flutter/src/gestures/recognizer.dart:182)
09-11 12:05:44.233 27550 27566 E flutter : #4      TapGestureRecognizer._checkUp (package:flutter/src/gestures/tap.dart:365)
09-11 12:05:44.233 27550 27566 E flutter : #5      TapGestureRecognizer.handlePrimaryPointer (package:flutter/src/gestures/tap.dart:275)
09-11 12:05:44.233 27550 27566 E flutter : #6      PrimaryPointerGestureRecognizer.handleEvent (package:flutter/src/gestures/recognizer.dart:467)
09-11 12:05:44.233 27550 27566 E flutter : #7      PointerRouter._dispatch (package:flutter/src/gestures/pointer_router.dart:75)
09-11 12:05:44.233 27550 27566 E flutter : #8      PointerRouter.route (package:flutter/src/gestures/pointer_router.dart:102)
09-11 12:05:44.233 27550 27566 E flutter : #9      GestureBinding.handleEvent (package:flutter/src/gestures/binding.dart:218)
09-11 12:05:44.233 27550 27566 E flutter : #10     GestureBinding.dispatchEvent (package:flutter/src/gestures/binding.dart:198)
09-11 12:05:44.233 27550 27566 E flutter : #11     GestureBinding._handlePointerEvent (package:flutter/src/gestures/binding.dart:156)
09-11 12:05:44.233 27550 27566 E flutter : #12     GestureBinding._flushPointerEventQueue (package:flutter/src/gestures/binding.dart:102)
09-11 12:05:44.233 27550 27566 E flutter : #13     GestureBinding._handlePointerDataPacket (package:flutter/src/gestures/binding.dart:86)
09-11 12:05:44.233 27550 27566 E flutter : #14     _rootRunUnary (dart:async/zone.dart:1136)
09-11 12:05:44.233 27550 27566 E flutter : #15     _CustomZone.runUnary (dart:async/zone.dart:1029)
09-11 12:05:44.233 27550 27566 E flutter : #16     _CustomZone.runUnaryGuarded (dart:async/zone.dart:931)
09-11 12:05:44.233 27550 27566 E flutter : #17     _invoke1 (dart:ui/hooks.dart:263)
09-11 12:05:44.233 27550 27566 E flutter : #18     _dispatchPointerDataPacket (dart:ui/hooks.dart:172)
09-11 12:05:44.233 27550 27566 E flutter : 
```

**Comment by tonymorony** [comment]
Fixed in 0.1.8
